internal/server/v1/page/golden/place_page_test.go

```diff
@@ -59,7 +59,7 @@ func TestPlacePage(t *testing.T) {
 				1,
```

Comment:
Do we need to handle v0 test file too?

Reply:
They are not used anymore; Removing the v0 code and golden tests.